JavaArgumentsOptimization.md

```diff
@@ -22,8 +22,10 @@ These flags are made by Aikar and you can find the explanation for the flags [he
 
 Use these flags exactly, only changing Xmx and Xms. These flags work and scale accordingly to any size of memory.
 
+If you're using the vanilla launcher, remove java from the start. Some other laumchers require java at the start
```

Comment:
`laumchers` please fix the spelling

Reply:
alright, I have made it more general, so it can apply to anything